tools/CMakeLists.txt

```diff
@@ -14,10 +17,14 @@ INCLUDE_DIRECTORIES(
 
 ADD_EXECUTABLE(mdalinfo ${MDALINFO_SOURCES})
 ADD_EXECUTABLE(mdal_translate ${MDAL_TRANSLATE_SOURCES})
+ADD_LIBRARY(mdal_dummy_external_driver SHARED ${MDAL_DUMMY_EXTERNAL_DRIVER})
```

Comment:
i think at the end dummy test driver should exist in self-contained CMakeLists.txt. also the external driver header and cpp should be separate folder. so the developers can copy-paste minimal dynamic driver and start from there

Reply:
i can help with cmake at the end